batch-nft-reveal/deploy/02_Deploy_NFTCollection.ts

```diff
@@ -47,22 +48,34 @@ const func: DeployFunction = async function ({
   const { gasLane: vrfGasLane, callbackGasLimit: vrfCallbackGasLimit } =
     networkConfig[chainId];
 
-  await deploy("NFTCollection", {
+  const args = [
+    NFT_NAME,
+    NFT_SYMBOL,
+    NFT_MAX_SUPPLY,
+    NFT_MINT_COST,
+    NFT_REVEAL_BATCH_SIZE,
+    NFT_REVEAL_INTERVAL,
+    vrfCoordinatorV2Address,
+    vrfSubscriptionId,
+    vrfGasLane,
+    vrfCallbackGasLimit,
+  ];
+
+  const result = await deploy("NFTCollection", {
```

Comment:
Maybe extract this string "NFTCollection" to another variable similar to deployment scripts template-nft-collection-erc721?